js/server/assets/swagger/index.html

```diff
@@ -1,132 +1,92 @@
+    <style>
+      html {
+        box-sizing: border-box;
+        overflow: -moz-scrollbars-vertical;
+        overflow-y: scroll;
+      }
 
-    <symbol viewBox="0 0 24 24" id="jump-to">
-      <path d="M19 7v4H5.83l3.58-3.59L8 6l-6 6 6 6 1.41-1.41L5.83 13H21V7z"/>
-    </symbol>
+      *,
+      *:before,
+      *:after {
+        box-sizing: inherit;
+      }
 
-    <symbol viewBox="0 0 24 24" id="expand">
-      <path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"/>
-    </symbol>
+      body {
+        margin:0;
+        background: #fff;
+        font-family: 'Open Sans', sans-serif;
+      }
 
-  </defs>
-</svg>
+      .topbar {
+        display: none;
+      }
 
-<div id="swagger-ui"></div>
+      .swagger-ui .scheme-container {
+        display: none;
+      }
 
-<script src="./swagger-ui-bundle.js"> </script>
-<script src="./swagger-ui-standalone-preset.js"> </script>
-<script>
-window.onload = function() {
-  var url = window.location.href.replace("index.html", "swagger.json").replace(/#.*/, "");
-  var jwt = sessionStorage.getItem("jwt");
-  
-  // Build a system
-  const ui = SwaggerUIBundle({
-    url: url,
-    dom_id: '#swagger-ui',
-    deepLinking: true,
-    tagsSorter: "alpha",
-    validatorUrl: null,
-    docExpansion: "list",
-    presets: [
-      SwaggerUIBundle.presets.apis,
-      SwaggerUIStandalonePreset
-    ],
-    plugins: [
-      SwaggerUIBundle.plugins.DownloadUrl
-    ],
-    layout: "StandaloneLayout",
-    requestInterceptor:
-      function (request) {
-        request.headers.Authorization = 'bearer ' + jwt;
-        return request;
+      .swagger-ui .info {
+        margin: 15px 0;
+      }
+      .swagger-ui a.nostyle span {
+        font-family: Open Sans, sans-serif;
+        font-weight: 400;
       }
-  })
 
-  window.ui = ui
-}
-</script>
-</body>
+      .swagger-ui pre,
+      .swagger-ui .markdown p {
+        font-weight: 100 !important;
+      }
 
+      .swagger-ui .markdown code p {
+        color: #fff !important;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+
+    <script src="./swagger-ui-bundle.js"> </script>
+    <script src="./swagger-ui-standalone-preset.js"> </script>
+    <script>
+    window.onload = function() {
+      var url = window.location.href.replace("index.html", "swagger.json").replace(/#.*/, "");
+      var jwt = sessionStorage.getItem("jwt");
+      // Begin Swagger UI call region
+      const ui = SwaggerUIBundle({
+        url: url,
+        dom_id: '#swagger-ui',
+        deepLinking: true,
+        presets: [
+          SwaggerUIBundle.presets.apis,
+          SwaggerUIStandalonePreset
+        ],
+        plugins: [
+          SwaggerUIBundle.plugins.DownloadUrl
+        ],
+        layout: "StandaloneLayout",
+        docExpansion: "list",
+        requestInterceptor: function (request) {
+          if (!request.headers.Authorization && !request.headers.authorization) {
```

Comment:
FYI this is a change in our code: instead of always setting this header we now only set it if not already present. This avoids issues later on caused by accidentally overriding the user-specified value.

Reply:
> This avoids issues later on caused by accidentally overriding the user-specified value.
Somehow we need to be sure, that we do not throw away that change in the future

Reply:
@pluma could we maybe write a dumb test which will fail if we update without keeping that change?

Reply:
I don't see how. This code is only executed if a user opens the swagger-ui in the browser and performs an operation. That'd require browser instrumentation to test, which we don't have and don't want.

The upstream swagger-ui code doesn't include the `requestInterceptor` config at all so updating without touching this would visibly break by making swagger-ui no longer load any specs at all and show a big scary error message.

The change my comment refers to is wrapping our code in an `if` to avoid clobbering user-provided headers (via the swagger-ui auth GUI).

As long as everyone who upgrades swagger-ui in our code follows the established practice, we'll be fine:

1. Find out which swagger-ui version we are currently using
2. Diff our index.html against the index.html bundled with that version by default
3. Replace our swagger-ui with the latest distribution
4. Manually re-apply the diff as appropriate

Yes, this is finicky, but this is the same process we've had for upgrading Swagger UI from the start.

Reply:
please add this procedure to https://github.com/arangodb/arangodb/blob/devel/3rdParty/README_maintainers.md

Reply:
@dothebart sounds good to me. I'll add it in this PR.